internal/pkg/build/sources/packer_sif_linux.go

```diff
@@ -45,6 +70,30 @@ func (p *SIFPacker) unpackSIF(b *types.Bundle, srcfile string) (err error) {
 		return fmt.Errorf("no root filesystem found in %s", srcfile)
 	}
 
+	// Open the SIF
+	fimg, err := sif.LoadContainer(srcfile, true)
+	if err != nil {
+		sylog.Fatalf("failed to load SIF container file: %s", err)
+	}
+	defer fimg.UnloadContainer()
+
+	sifData, _, err := getMetaData(&fimg, sif.DataLabels)
+	if err == ErrNoMetaData {
+		sylog.Warningf("No metadata partition found")
+	} else if err != nil {
+		sylog.Fatalf("Unable to get label metadata: %s", err)
+	} else if err == nil {
+		b.Recipe.ImageData.Labels = make(map[string]string, 1)
```

Comment:
Lets not store this within the recipe, rather the `bundle.JSONObjects` map, this looks like it will overwrite labels that exist within a definition right?

Reply:
Actually, the labels from the deffile will override these labels. Also, I tried to use the `JSONObjects` map, but I had a hard time getting that to work...